packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -207,6 +207,7 @@ function toTree(vnode) {
     case FiberTags.ContextProvider:
     case FiberTags.ContextConsumer:
       return childrenToTree(node.child);
+    case FiberTags.Profiler:
```

Comment:
This was the mistake. I previously included this in the same branch as the context tags, strict mode etc. That branch will not be considered a node. At least this is how I understood it.

There's probably an argument to be made that they should also be considered as valid nodes (since they also appear in reacts component stack and devtools) but that's for another time (personally I'm not interested in those components WRT to find/debug).

Not sure if Profiler should be handled *exactly* like `forwardRef` but tests are passing for now so :man_shrugging:

Reply:
hmm - if they appear in react devtools, and the stack, then they *definitely* need to appear in `debug`.